control-plane/modules/aws-k3s/files/rancher-install.sh

```diff
@@ -5,15 +5,13 @@ kubectl taint nodes $node monitoring=yes:NoSchedule
 kubectl label nodes $node monitoring=yes
 
 %{ if install_certmanager ~}
-kubectl create namespace cert-manager
-kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
```

Comment:
Do we not need to do this anymore?

Reply:
Nope, this is pre 1.0 stuff, current install instructions: https://cert-manager.io/v1.5-docs/installation/kubectl/